doc/release/1.16.0-notes.rst

```diff
@@ -6,6 +6,8 @@ NumPy 1.16.0 Release Notes
 Highlights
 ==========
 
+* Python 3.4 support has officially been dropped.
```

Comment:
Not sure we need the word "official" in the release notes which are distributed "officially" anyway.